### PR TITLE
Fix RHBugzilla instantiation as new version requires url to be passed

### PR DESCRIPTION
**Pull request by rochacbruno**
The PR #4308 is pending this

### CODE REVIEW

File: tests/robottelo/test_decorators.py [hunk 1]
```diff
@@ -506,32 +510,10 @@ def test_not_cached_bug(self):
                 'robottelo.decorators._bugzilla', {}):
             self.assertEqual(id(decorators._get_bugzilla_bug('4242')), id(bug))
         self.bugzilla.RHBugzilla.assert_called_once_with(
+            url=BUGZILLA_URL,
             user=self.settings.username,
             password=self.settings.password
         )
-        connection.connect.assert_called_once_with(decorators.BUGZILLA_URL)
-
-    def test_raise_bug_fetch_error_connect_type_error(self):
```
Comment (rochacbruno) [inline tests/robottelo/test_decorators.py]: IN newer bugzilla no need to use `connect` so no need to test it!

**Comment by ldjebran** [comment]
@rochacbruno , perhaps we have run one test for example that should be skipped because of bug open and one with bug closed that should pass.
what do you think?

**Comment by rochacbruno** [comment]
@ldjebran @pondrejk 

```bash
brocha@localhost(rob63) :~/P/robottelo|fix_bugzilla_url✓
➤ py.test tests/foreman/api/test_contentview.py -v -k test_negative_update_label
======== test session starts =======
platform linux2 -- Python 2.7.11, pytest-3.0.6, py-1.4.32, pluggy-0.4.0 -- /home/brocha/.virtualenvs/rob63/bin/python2
cachedir: .cache
rootdir: /home/brocha/Projects/robottelo, inifile: 
plugins: xdist-1.15.0, cov-2.4.0
collected 57 items 

tests/foreman/api/test_contentview.py::ContentViewUpdateTestCase::test_negative_update_label <- robottelo/decorators/__init__.py SKIPPED

========== 56 tests deselected ===========================
========== 1 skipped, 56 deselected in 16.86 seconds =========


```

**Comment by ldjebran** [comment]
merge as requested only two reviewers

**Pull request merged**